### ISSUE TITLE
Extend arrival/departure dates to all users, not just instructors

### ISSUE THREAD

**Issue by maco** [skandragon/thing#112]
If all users (not just instructors) can put in their arrival/departure dates, then when they're building a custom schedule, the ones that occur only while the user is not present should be filtered out (optionally?).

**Comment by skandragon** [comment]
As you're the only coder so far who I've run into, can we talk at or before Pennsic about having you join the team we currently have?  With you, we'd have two coders!  :)

**Comment by maco** [comment]
Yep! Which that's weird because the SCA is full of programmers. Maybe
adding the "Fork this on Github" banner to the site would help there. I'm
Grazia and camp with Clan Cambion on Howard's Fenway (camp device is:
sable, a bend sinister argent, within a bordure sable and argent compony).

**Comment by skandragon** [comment]
Awesome.  I'm Michael Graff on Facebook; perhaps we can chat there a bit first?  I'm very excited to have found a second coder!  Also, I want to rewrite it all in meteor.js, but don't tell Artie. :)

**Comment by Artimisia** [comment]
So long as it works - I am happy.  If it works and looks the same (except, of course, for the changes I've asked for)-that is even better. :). Seriously, I will support in any way I can.

Artie

**Comment by maco** [comment]
I haven't used meteor.js or any js frameworks yet (but I'll be learning one
right after Pennsic for my new job). Rails and Django are my usual web
frameworks.
